File: CS/StandardErrorBars/Form1.cs
using DevExpress.XtraCharts;
using System;
using System.Collections.Generic;
using System.Windows.Forms;

namespace ErrorBarsWin {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }
        public class Record {
            public Record(DateTime timePoint, int temperature) {
                this.TimePoint = timePoint;
                this.Temperature = temperature;
            }
            public DateTime TimePoint { get; set; }
            public double Temperature { get; set; }
        }
        List<Record> GetData() {
            return new List<Record>() {
                new Record(new DateTime(2016, 07, 01), 26),
                new Record(new DateTime(2016, 07, 02), 24),
                new Record(new DateTime(2016, 07, 03), 26),
                new Record(new DateTime(2016, 07, 04), 29),
                new Record(new DateTime(2016, 07, 05), 30),
                new Record(new DateTime(2016, 07, 06), 33),
                new Record(new DateTime(2016, 07, 07), 32),
                new Record(new DateTime(2016, 07, 08), 30),
                new Record(new DateTime(2016, 07, 09), 22),
                new Record(new DateTime(2016, 07, 10), 27),
                new Record(new DateTime(2016, 07, 11), 27),
                new Record(new DateTime(2016, 07, 12), 28),
                new Record(new DateTime(2016, 07, 13), 29),
                new Record(new DateTime(2016, 07, 14), 31),
                new Record(new DateTime(2016, 07, 15), 31),
                new Record(new DateTime(2016, 07, 16), 32),
                new Record(new DateTime(2016, 07, 17), 32),
                new Record(new DateTime(2016, 07, 18), 34),
                new Record(new DateTime(2016, 07, 19), 28),
                new Record(new DateTime(2016, 07, 20), 29),
                new Record(new DateTime(2016, 07, 21), 32),
                new Record(new DateTime(2016, 07, 22), 34),
                new Record(new DateTime(2016, 07, 23), 36),
                new Record(new DateTime(2016, 07, 24), 34),
                new Record(new DateTime(2016, 07, 25), 34),
                new Record(new DateTime(2016, 07, 26), 32),
                new Record(new DateTime(2016, 07, 27), 33),
                new Record(new DateTime(2016, 07, 28), 35),
                new Record(new DateTime(2016, 07, 29), 29),
                new Record(new DateTime(2016, 07, 30), 29),
                new Record(new DateTime(2016, 07, 31), 26)
            };
        }
        Series series { get { return chartControl.Series[0]; } }
        #region #StandardErrorBars
        private void FormOnLoad(object sender, EventArgs e) {
            series.ArgumentDataMember = "TimePoint";
            series.ValueDataMembers.AddRange(new string[] { "Temperature" });
            series.DataSource = GetData();
            PointSeriesView view = (PointSeriesView)chartControl.Series[0].View;
            view.Indicators.Add(new StandardErrorBars {
                Direction = ErrorBarDirection.Both,
                Name = "Standard Error",
                ShowInLegend = true
            });
        }
        #endregion #StandardErrorBars
    }
}

File: VB/StandardErrorBars/Form1.vb
Imports DevExpress.XtraCharts
Imports System
Imports System.Collections.Generic
Imports System.Windows.Forms

Namespace ErrorBarsWin
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub
        Public Class Record
            Public Sub New(ByVal timePoint As Date, ByVal temperature As Integer)
                Me.TimePoint = timePoint
                Me.Temperature = temperature
            End Sub
            Public Property TimePoint() As Date
            Public Property Temperature() As Double
        End Class
        Private Function GetData() As List(Of Record)
            Return New List(Of Record)() From { _
                New Record(New Date(2016, 07, 01), 26), _
                New Record(New Date(2016, 07, 02), 24), _
                New Record(New Date(2016, 07, 03), 26), _
                New Record(New Date(2016, 07, 04), 29), _
                New Record(New Date(2016, 07, 05), 30), _
                New Record(New Date(2016, 07, 06), 33), _
                New Record(New Date(2016, 07, 07), 32), _
                New Record(New Date(2016, 07, 08), 30), _
                New Record(New Date(2016, 07, 09), 22), _
                New Record(New Date(2016, 07, 10), 27), _
                New Record(New Date(2016, 07, 11), 27), _
                New Record(New Date(2016, 07, 12), 28), _
                New Record(New Date(2016, 07, 13), 29), _
                New Record(New Date(2016, 07, 14), 31), _
                New Record(New Date(2016, 07, 15), 31), _
                New Record(New Date(2016, 07, 16), 32), _
                New Record(New Date(2016, 07, 17), 32), _
                New Record(New Date(2016, 07, 18), 34), _
                New Record(New Date(2016, 07, 19), 28), _
                New Record(New Date(2016, 07, 20), 29), _
                New Record(New Date(2016, 07, 21), 32), _
                New Record(New Date(2016, 07, 22), 34), _
                New Record(New Date(2016, 07, 23), 36), _
                New Record(New Date(2016, 07, 24), 34), _
                New Record(New Date(2016, 07, 25), 34), _
                New Record(New Date(2016, 07, 26), 32), _
                New Record(New Date(2016, 07, 27), 33), _
                New Record(New Date(2016, 07, 28), 35), _
                New Record(New Date(2016, 07, 29), 29), _
                New Record(New Date(2016, 07, 30), 29), _
                New Record(New Date(2016, 07, 31), 26) _
            }
        End Function
        Private ReadOnly Property series() As Series
            Get
                Return chartControl.Series(0)
            End Get
        End Property
        #Region "#StandardErrorBars"
        Private Sub FormOnLoad(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            series.ArgumentDataMember = "TimePoint"
            series.ValueDataMembers.AddRange(New String() { "Temperature" })
            series.DataSource = GetData()
            Dim view As PointSeriesView = CType(chartControl.Series(0).View, PointSeriesView)
            view.Indicators.Add(New StandardErrorBars With {.Direction = ErrorBarDirection.Both, .Name = "Standard Error", .ShowInLegend = True})
        End Sub
        #End Region ' #StandardErrorBars
    End Class
End Namespace